src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/hmi_command_factory.cc

```diff
@@ -547,6 +550,9 @@ CommandCreator& HMICommandFactory::get_creator_factory(
     case hmi_apis::FunctionID::BasicCommunication_OnAppActivated: {
       return factory.GetCreator<commands::OnAppActivatedNotification>();
     }
+    case hmi_apis::FunctionID::SDL_GetDeviceConnectionStatus: {
```

Comment:
@ZhdanovP I see notification, but where is `GetDeviceConnectionStatusRequest` and `GetDeviceConnectionStatusResponse`?

Reply:
@AKalinich-Luxoft please take a look the latest commits.